gateway/gateway-runtime/Makefile

```
@@ -0,0 +1,127 @@
+DOCKER_REGISTRY ?= ghcr.io/wso2/api-platform
+IMAGE_NAME := $(DOCKER_REGISTRY)/gateway-runtime
+
+.PHONY: help build build-local build-coverage-image push clean build-and-push-multiarch
+
+help: ## Show this help message
+	@echo 'Gateway Runtime Build (Version: $(VERSION))'
+	@echo ''
+	@echo 'Available targets:'
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "  %-25s %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+build: ## Build Gateway Runtime Docker image using buildx
```

checkmake (0.2.2)

[warning] Target body for "build" exceeds allowed length of 5 (18).

(maxbodylength)